### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Shipow**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,28 +10,32 @@ See features and benefits of [Algolia Search Extension for Magento](https://comm
 ![Magento 1.9.2](https://img.shields.io/badge/magento-1.9.2-blue.svg)
 ![Magento 1.9.3](https://img.shields.io/badge/magento-1.9.3-blue.svg)
 
-Documentation
---------------
+- **Auto-completion menu:** Offer End-Users immediate access to your whole catalog from the dropdown menu, whatever your number of categories or attributes.
```
Comment (JanPetr) [inline README.md]: Don't you miss "is" on the end of the sentence?

Comment (Shipow) [inline README.md]: I can't see where but it's possible :)

Comment (JanPetr) [inline README.md]: Sentence "... whatever your number of categories or attributes." misses the verb. I guess it should be "... whatever your number of categories or attributes is.", no?

Comment (Shipow) [inline README.md]: it's "offer ..., whatever ..." so I think we are good.

Comment (JanPetr) [inline README.md]: Yeah, but "offer" is related to "immediate access" and not to "number of ..." :))
I checked in #help-aglophone and "is" should be there. I fixed that ;)

Comment (JanPetr) [inline README.md]: My bad, in the end the "is" shouldn't be there :D Will merge asap

**Pull request merged**